HULKs/hulk
# Reduce liquidity

## Why? What?

Remove superfluous parameters and add a check.

Fixes #

## ToDo / Known Issues

## Ideas for Next Iterations (Not This PR)

## How to Test

Add an unused field anywhere in the default.json or even a robot/location specific config file and upload.
Should error if the modified config file is actually used by the robot.
